moya/numbers.py

```diff
@@ -32,8 +32,13 @@ def csv_reader(fh, batch_size=1000):
     buf = []
     for row in csv.DictReader(fh):
         number = row['to']
+        
         if not is_number(number):
-            raise Exception("Line {number} doesn't look like a phone number")
+            continue
```

Comment:
keep existing logic please. if you want to make it skip instead then add a flag option and pass it in